Remote/Scripts/ClientGUI.cs

```diff
@@ -61,7 +61,10 @@ class ClientGUI : MonoBehaviour
         void Awake()
         {
             m_Camera = Camera.main;
-            if (Device.generation == DeviceGeneration.iPhoneX)
+            if (Device.generation == DeviceGeneration.iPhoneX || 
+                Device.generation == DeviceGeneration.iPhoneXS || 
```

Comment:
It looks like these defines were introduced somewhere in the 2018.2 cycle. Do you know what specific version is the earliest that supports this? I'd like to introduce an #if UNITY_2018_2_X_OR_NEWER so that this will still compile on older versions.

Reply:
hi! Thanks for considering my PR. Support for these devices was introduced on Unity 2018.2.12f1 as stated here https://forum.unity.com/threads/ios-12-xcode-10-iphone-xs-xr-support-status.560761/

Reply:
I intuit what you want to do, but not sure how to do it. Is there a list of existing constants like `UNITY_2018_2_X_OR_NEWER` or is it a custome defined constant? I can't find a proper value for the mentioned Unity version at https://docs.unity3d.com/Manual/PlatformDependentCompilation.html

Reply:
If we just want to know whether face tracking is supported on this device, UnityARKit plugin has a method for that:
ARKitFaceTrackingConfiguration.IsSupported()

Reply:
If we just want to know whether face tracking is supported on this device, UnityARKit plugin has a method for that:
ARKitFaceTrackingConfiguration.IsSupported()

Reply:
f we just want to know whether face tracking is supported on this device, UnityARKit plugin has a method for that:
ARKitFaceTrackingConfiguration.IsSupported()